README.md

```diff
@@ -229,6 +229,15 @@ During the current, early stage of development using a higher, cross-compilation
 system such as Yocto is advisable. Once things are sufficiently stabilized, a set of steps for
 manual cross-compilation will be added here.
 
+## QNX
+
+```
+mkdir build && cd build
```

Comment:
S:
```
cmake -S . -B build && QNX_TARGET=<compiler-selection> cmake --build ./build -D CMAKE_TOOLCHAIN_FILE=cmake/qnx.cmake
```

Reply:
On second thought I think I will leave it as is. We do the `cmake ..` variant on Linux, and if we don't do `cd build`, then `make` doesn't work.

Reply:
Sure fine, just a suggestion. `cmake --build build` has one benefit over `make` however, and that is that it runs your build system, whether it is `make` or not. So if your default `buildsystem` is not make, this will still work